app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError(f"{visitor['name']} is not vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError(
+                f"{visitor['name']} has an expired vaccination"
```

Comment:
Ensure that the error message is descriptive. Consider including more context in the message, such as specifying that the visitor's vaccination is expired and they cannot enter.